modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/metadata/MetadataOutputDelegate.java

```diff
@@ -140,6 +143,9 @@
           .withReason(NULL_TYPE_ERROR)
           .onOutputPayload();
       return failure(output.getType(), failure);
+    } catch (ConnectionException e) {
+      return failure(newFailure(e).withMessage("Failed to establish connection: " + ExceptionUtils.getMessage(e))
+          .withFailureCode(CONNECTION_FAILURE).onKeys());
```

Comment:
same here

Reply:
Good catch. Yes, it should be onComponent().